### DISCUSSION TITLE
GitHub Packages versions are missing version numbers

### DISCUSSION THREAD

**Discussion by bobwang104** [community/community#135750] [Packages] [answered]
### Select Topic Area

Question

### Body

In the GitHub Packages web UI, version numbers or names were originally displayed, but they have suddenly disappeared. How can I restore them?

Before
![image](https://github.com/user-attachments/assets/08f0933d-4dd0-42f6-98b4-f32a5daa07a3)

After
![image](https://github.com/user-attachments/assets/a615b8b0-d8f8-4fcc-8a28-1a0b270505fc)

**Comment by chizkiyahu** [comment]
I also have the same problem 
Is very annoying and make hard to debug problems

**Reply by yuvalavr24** [reply]
Same.

**Reply by yoelp2k** [reply]
same

**Reply by chenxin6** [reply]
same

**Reply by nlisgo** [reply]
Same. It's unusable at the moment, for me.

**Comment by lulzactive** [comment]
same

**Comment by RyStanley89** [comment, marked as answer]
Hello all 👋 Just to let you know that this was a bug which is now fixed. Please check again and let us know if the tags still aren't present.
